projects/kit/components/input-password/input-password.template.html

```diff
@@ -25,6 +25,7 @@
         class="icon"
         tuiHintId="not_required"
         [tuiHint]="hintContent"
+        [tuiHintMode]="options.hintMode"
```

Comment:
Are you sure we need this here? Maybe it should just rely on `TuiMode` directive, like regular input field hint?

Reply:
I think you are right. I've changed it to a hint controller